tests/Temporalio.Tests/Worker/WorkflowWorkerTests.cs

```diff
@@ -4449,7 +4448,58 @@ await foreach (var evt in handle2.FetchHistoryEventsAsync())
             worker.Client = otherEnv.Client;
 
             // Now confirm the other workflow has started
-            await AssertHasEventEventuallyAsync(handle1, e => e.WorkflowTaskCompletedEventAttributes != null);
+            await handle2.AssertHasEventEventuallyAsync(e => e.WorkflowTaskCompletedEventAttributes != null);
```

Comment:
I think this was a typo from the original PR? I changed the test case to check `handle2` instead of `handle1`.

Reply:
Yes, thanks!

Reply:
nope, tests passed fine!